code/datums/supplypacks.dm

```diff
@@ -1792,6 +1793,26 @@ AMMO
 	containername = "\improper incendiary sniper ammo crate"
 	group = "Ammo"
 
+/datum/supply_packs/ammo_incin_M41A1
```

Comment:
We are replacing the M41A1 soon, so this change will become outdated.
The new gun won't have incendiary bullets.

Reply:
Cool, I'll be sure to add in incendiary bullets for it so you can ree at me about balance implications, I can accept and remove it from in-game access, and admemes get a fun thing to spawn for EORG.

Reply:
don't use upper case typepaths